docs/source/_contents/ipyleaflet.ipynb

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I guess you did not want to commit this file. We should probably add `docs/source/_contents` to the `.gitignore`